server/api/checkout.js

```diff
@@ -15,25 +15,23 @@ const router = express();
 //     })
 
 //     console.log('client secret', client_secret)
-  
+
 //     res.send(client_secret);
 // });
 
 router.get('/card-wallet', async (req, res) => {
+  const customer = await stripe.customers.create();
 
-    const customer = await stripe.customers.create();
-
-    const intent =  await stripe.setupIntents.create({
-      customer: customer.id,
-    });
+  const intent = await stripe.setupIntents.create({
+    customer: customer.id,
+  });
 
-    res.send('card_wallet', { client_secret: intent.client_secret });
+  res.send('card_wallet', { client_secret: intent.client_secret });
 });
 
-const paymentMethods = await stripe.paymentMethods.list({
-    customer: '{{CUSTOMER_ID}}',
-    type: 'card',
-}); 
-
+// const paymentMethods = await stripe.paymentMethods.list({
+//     customer: '{{CUSTOMER_ID}}',
+//     type: 'card',
+// });
```

Comment:
this can all come out.. my bad, I was just setting up for the paymentIntent (it's all set up in my current PR)